src/ddmd/dclass.d

```diff
@@ -212,20 +234,32 @@ extern (C++) class ClassDeclaration : AggregateDeclaration
     // the ClassInfo object for this ClassDeclaration
     TypeInfoClassDeclaration vclassinfo;
 
-    bool com;           // true if this is a COM class (meaning it derives from IUnknown)
-    bool cpp;           // true if this is a C++ interface
-    bool isobjc;        // true if this is an Objective-C class/interface
-    bool isscope;       // true if this is a scope class
-    private bool inuse;          // to prevent recursive attempts
+    // true if this is a COM class
+    bool com;
+
+    /// true if this is a scope class
+    bool stack;
+
+    /// specifies whether this is a d, cpp, objc or anonymous class
```

Comment:
Nitpick:

* I think it looks better to use the official names of the languages in the documentation, i.e. `D`, `C++` and `Objective-C`
* `class` -> `class/interface`